### ISSUE TITLE
Implement fidgets

### ISSUE THREAD

**Issue by methusalah** [methusalah/OpenRTS#4]
Unit may move time to time to add some life to the battleground

**Comment by wuendsch** [comment]
it's look like the command "patrol" in Starcraft 2, isn't it? http://starcraft.wikia.com/wiki/Patrol

**Comment by methusalah** [comment]
No,  fidgets are the little movements (mainly rotations, or even
animations) units and unit's turrets do spontaneously to add life to the
battlefield

Le sam. 16 mai 2015 10:20, Mario Meltzow <notifications@github.com> a
écrit :
